src/main.rs

```diff
@@ -342,8 +349,67 @@ impl Display for MenuItem {
     }
 }
 
+fn write_undo_renames(backup_file: PathBuf, replacements: Vec<Rename>) -> anyhow::Result<()> {
+    let undo_replacements = replacements
+        .into_iter()
+        .map(|r| {
+            // make paths absolute to that undo does not depend on CWD
+            let original = if r.original.is_relative() {
+                std::env::current_dir()?.join(r.original)
+            } else {
+                r.original
+            };
+            let new = if r.new.is_relative() {
+                std::env::current_dir()?.join(r.new)
+            } else {
+                r.new
+            };
```

Comment:
could you make a function for converting the relative path to an absolute one so we don't repeat here?